gapis/api/vulkan/frame_loop.go

```diff
@@ -643,3 +708,21 @@ func (f *frameLoop) copyImage(ctx context.Context, srcImg, dstImg ImageObjectʳ,
 
 	return nil
 }
+
+func (f *frameLoop) resetDescriptorSetLayouts(ctx context.Context, stateBuilder *stateBuilder) error {
+
+	// For every DescriptorSetLayout that was created during the loop...
+	for created, _ := range f.descriptorSetLayoutCreated {
```

Comment:
Maybe `for created := range f.descriptorSetLayoutCreated {`

Reply:
Good catch @RenfengLiu. You can just drop the _.